mentors/src/data_pipeline/ibm_transcript_service.py

```diff
@@ -9,14 +10,16 @@
 Contact Madhusudhan Krishnamachari at madhusudhank@icloud.com to get assistance on how to use Watson. (IBM Documentation should help)
 """
 passphrase = ""
-with open("../../website_version/password.txt", "r") as password:
+with open("src/data_pipeline/secrets/password.txt", "r") as password:
     passphrase = password.read()
     print(passphrase)
 speech_to_text = SpeechToTextV1(
-    username="b339aacb-e633-4e40-b10d-6f5b300f59bf",
-    password=passphrase,
-    x_watson_learning_opt_out=True,  # tells IBM Watson not to collect our data, hence keeping our data confidential and secure.
+    username="b339aacb-e633-4e40-b10d-6f5b300f59bf", password=passphrase
```

Comment:
let's get username out of the code here. Probably should live in an uncommitted config file as `password.txt` already does

Reply:
Okay, we may need to wipe some git history

Reply:
I wouldn't worry about it, it will get garbage collected after squash/merge and not super sensitive info

Reply:
Its been in the history for 11 months so squashing here probably won't help. https://github.com/benjamid/MentorPAL/commit/e775845eb4c2c2081974bf50ed31e2feb9b4b2f6

Regardless I'll move it now.

Reply:
yeah, i'm not too worried about the security implications. Mainly there's just no reason to make the code statically locked to a specific account, especially when no one could use it anyway unless they had a local `password.txt` file. It's all downside, no upside